internal/pkg/mount/mount.go

```diff
@@ -18,7 +21,27 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-// Point represents a linux mount point.
+// RetryFunc defines the requirements for retrying a mount point operation.
+type RetryFunc func(*Point) error
```

Comment:
Adding overlay functionality triggered the `gocyclo` linter, and, in this case, I agreed with it. This function type was introduced to make retries more generic in order to make things more DRY, and thereby reducing the complexity.